notebooks/1-lightcurves/TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1265 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    filtered.show_in_notebook(display_length=10)
</pre><p>Is <code>display_length</code> supposed to limit the number of rows shown to 10? It doesn't appear to be doing this. We may want to just remove this argument.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>removed argument, limited results to show only first 10 rows though</p>